m2c_helper.sh

```diff
@@ -45,11 +45,11 @@ COMMON_CONTEXT="ctx.c"
 # Extra options when generating the function specific context
 CONTEXT_M2C_OPTS="--stack-structs --zfill-constants"
 # Extra options when generating the function disassembly
-DISASEMBLY_M2C_OPTS="--zfill-constants"
+DISASSEMBLY_M2C_OPTS="--zfill-constants"
 # Directories to search when a .incbin line is encountered in a data file
 INCBIN_OPTS="--incbin-dir ./expected/build/us --incbin-dir ./"
 # File that the final disassebmly output will be placed. Can be named whatever you want
-DISASEMBLY_FILE="function_disassembly.c"
+DISASSEMBLY_FILE="tyler.c"
```

Comment:
Is tyler.c on purpose? (if it is it's fine)

Reply:
Nope, that was a mistake on my part. I name it after myself for my own purposes but for general use I want it named something else. There's no reason to use my name as the default.